src/lib/Menu/ContentEditRightSidebarBuilder.php

```diff
@@ -75,24 +94,39 @@ public function createStructure(array $options): ItemInterface
         /** @var Location $parentLocation */
         $parentLocation = $options['parent_location'];
 
+        $canPublish = $this->permissionResolver->canUser('content', 'publish', $content);
+        $canEdit = $this->permissionResolver->canUser('content', 'edit', $content);
+        $canDelete = $this->permissionResolver->canUser('content', 'remove', $content);
+
+        $publishAttributes = [
+            'class' => 'btn--trigger',
```

Comment:
You could store the reference to `btn-trigger` CSS class name in some kind of variable, to avoid value duplication.